tests/phpunit/tests/html-api/wpHtmlTagProcessor-internals.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Unit tests covering WP_HTML_Tag_Processor internal helpers.
+ *
+ * @package WordPress
+ * @subpackage HTML-API
+ */
+
+/**
+ * @group html-api
+ *
+ * @coversDefaultClass WP_HTML_Tag_Processor
+ */
+class Tests_HtmlApi_WpHtmlTagProcessor_Internals extends WP_UnitTestCase {
+	/**
+	 * @ticket {TICKET NUMBER}
+	 *
+	 * @dataProvider data_html_with_entire_tag_raw_markup
+	 *
+	 * @param string $html_with_target_element HTML with a tag containing "target" attribute.
+	 * @param string $expected_raw_markup      Expect full raw markup for targeted tag.
```

Comment:
```suggestion
	 * @param string $expected_raw_markup      Expected full raw markup for targeted tag.
```

(?)

Reply:
Updated in 7151083918 - thank you, and yes, your hunch was the right one.